frontend/cypress/e2e/navbar.cy.js

```diff
@@ -0,0 +1,68 @@
+//cy.signup("name", "surname", "someone@example.com", "password");
+
+
+describe('Navbar', () => {
+  it('shows navbar once user has signed in', () => {
+    cy.visit("/signup")
+    cy.get('nav').should('not.exist')
+    cy.signup("name", "surname", "someone@example.com", "password");
+
+    cy.visit("/login");
+    cy.get('nav').should('not.exist')
+    cy.get("#email").type("someone@example.com");
+    cy.get("#password").type("password");
+    cy.get("#submit").click();
+    cy.wait(1000)
```

Comment:
[❓ ] Why do we need a wait here?

Reply:
This is a wait of 1 second after posting the login details to allow the request time to process before continuing with the test. This prevents/reduces errors in the tests from async issues.